src/frontend/packages/suse-extensions/src/custom/helm/store/helm.effects.ts

```diff
@@ -76,25 +130,41 @@ export class HelmEffects {
   @Effect()
   fetchCharts$ = this.actions$.pipe(
     ofType<GetMonocularCharts>(GET_MONOCULAR_CHARTS),
-    flatMap(action => {
+    withLatestFrom(this.store),
+    flatMap(([action, appState]) => {
       const entityKey = entityCatalog.getEntityKey(action);
-      return this.makeRequest(action, `/pp/${this.proxyAPIVersion}/chartsvc/v1/charts`, (response) => {
-        const base = {
-          entities: { [entityKey]: {} },
-          result: []
-        } as NormalizedResponse;
 
-        const items = response.data as Array<any>;
-        const processedData = items.reduce((res, data) => {
-          const id = getMonocularChartId(data);
-          res.entities[entityKey][id] = data;
-          // Promote the name to the top-level object for simplicity
-          data.name = data.attributes.name;
-          res.result.push(id);
-          return res;
-        }, base);
-        return processedData;
-      }, []);
+      this.store.dispatch(new StartRequestAction(action));
+
+      const helmEndpoints = Object.values(endpointOfTypeSelector(HELM_ENDPOINT_TYPE)(appState));
+      const helmHubEndpoint = helmEndpoints.find(endpoint => endpoint.sub_type === HELM_HUB_ENDPOINT_TYPE);
+
+      const stratosMonocular = this.httpClient.get<MonocularChartsResponse>(`/pp/${this.proxyAPIVersion}/chartsvc/v1/charts`);
```

Comment:
Create Issue - investigate a way to use the cnis-list header to go out to all helm types